src/screens/Documents/View/index.js

```diff
@@ -101,16 +102,27 @@ class DocumentsViewScreen extends NavigatorComponent<Props & DocumentsState & Ac
   };
 
   onSelectDelete = () => {
+    Alert.alert(
+      'Confirm Delete',
```

Comment:
Please use i18n for text literals. You can also consider use [`alert` function](https://github.com/Bit-Nation/BITNATION-Pangea-mobile/blob/develop/src/global/alerts.js#L41) to benefit from the generalization. That way you don't need to deal much with i18n. You can find example usage of `alert` throughout the app.

Reply:
Done